app/helpers/application_helper.rb

```diff
@@ -36,4 +36,20 @@ def alert_class(alert_type)
     }.with_indifferent_access.fetch(alert_type, alert_type.to_s)
     "alert-#{alert_type}"
   end
+  def react_component(component_name, props={}, html_options = {}, render_order: 0)
```

Comment:
That totally works.  I recall some helper like this for angular for which someone had written a Gem but then changes to Sprockets messed it up.  it seemed about as simple to maintain your own helper method as to figure out that the Gem is not maintained or needs an update.